OptimoveSDK/Sources/Classes/Optimobile/InApp/InAppPresenter.swift

```diff
@@ -278,9 +278,11 @@ class InAppPresenter: NSObject, WKScriptMessageHandler, WKNavigationDelegate {
         #else
             let cachePolicy = URLRequest.CachePolicy.reloadIgnoringCacheData
         #endif
-        let url = Optimobile.getInstance().urlBuilder.urlForService(.iar)
-        let request = URLRequest(url: URL(string: url)!, cachePolicy: cachePolicy, timeoutInterval: 8)
-        webView.load(request)
+        if let urlString = KeyValPersistenceHelper.object(forKey: OptimobileUserDefaultsKey.IAR_BASE_URL.rawValue) as? String,
```

Comment:
Is there a reason we persist IAR base URL? It is constant & non-regionalised for production.

(and sorry, appreciate I'm late to the party, there will be context I'm missing, just curious--happy to discuss next week!)

Reply:
Thank you for your comment. 
Indeed, the IAR_BASE_URL is a constant and we can skip passing it through the storage layer.